starter-code/index.html

```diff
@@ -3,40 +3,88 @@
 <head>
   <meta charset="UTF-8">
   <title>Spotify</title>
+  <link rel="stylesheet" href="./css/main.css">
 </head>
 <body>
-Premium Discover Help Download
-
-Music for everyone.
-Spotify is now free on mobile, tablet and computer.
-Listen to the right music, wherever you are.
-
-What’s on Spotify?
-
-Millions of Songs
-There are millions of
-songs on Spotify
-
-HD Music
-Listen to music as if you
-were listening live
-
-Stream Everywhere
-Stream music on your
-smartphone, tablet or
-computer
-
-It’s as yeezy as Kanye West.
-Search
-Know what you want to listen to?
-Just search and hit play.
-Browse
-Check out the latest charts,
-brand new releases and great
-playlists for right now.
-Discover
-Enjoy new music every Monday
-with your own personal playlist.
-Or sit back and enjoy Radio.
-</body>
-</html>
+
+  <section id="nav-bar">
+    <nav>
+      <div id="logo"> 
+        <img src="./images/spotify-logo.png" alt="spotify-logo">
+      </div>
+      <div id="menu">
+        <ul>
+          <li><a href="#">Premium</a></li>
+          <li><a href="#">Discover</a></li>
+          <li><a href="#">Help</a></li>
+          <li><a href="#">Download</a></li>
+        </ul>
+      </div>
+    </nav>
```

Comment:
The `nav` tag eliminates the need for a `ul`-`li` structure.  You can simply pop your links inside the `nav` without the `li`'s.  Then I would take out the logo from the `nav` and have the `nav` and the `logo` be siblings.